bin/deploy.sh

```diff
@@ -0,0 +1,28 @@
+#! /bin/bash
+
+# Tag, Push and Deploy only if it's not a pull request
+if [ -z "$TRAVIS_PULL_REQUEST" ] || [ "$TRAVIS_PULL_REQUEST" == "false" ]; then
+
+    # Push only if we're testing the staging branch
+    #if [ "$TRAVIS_BRANCH" == "staging" ]; then
+
+        export PATH=$PATH:$HOME/.local/bin
+
+        echo Getting the ECR login...
+        eval $(aws ecr get-login --region $AWS_DEFAULT_REGION)
+        
+        echo Running docker push command...
+        docker push "$DOCKER_REPO"/"$DEPLOY_TARGET"/"$DOCKER_IMAGE":latest
+        
+        #echo Running ecs-deploy.sh script...
```

Comment:
We ought to enable this by default as well, so that users don't have to figure out why containers aren't making it to ECS

Reply:
The deploy.sh copy is not a good example. I figured since you were testing that and had a working example you could add that in here.

Reply:
The script otherwise conforms to last year's approach which seemed to work out well enough, so I’d say we can start here and adjust if folks have better ideas.

Reply:
I didn't re-enable the call to bin/ecs-deploy.sh because I don't have a copy in the repo. I looked at the example you want to copy from, but it looks large and complex so I would feel more comfortable getting everything else working besides it and then spend some time looking at the script to get more comfortable before adding it myself. 

Alternatively someone more comfortable in that area could add it.

Reply:
ok will leave it there